contentcuration/contentcuration/frontend/channelEdit/pages/StagingTreeView/index.vue

```diff
@@ -1,229 +1,248 @@
-              {{ $tr('totalResources') }}:
-              <span class="font-weight-bold">{{ resourcesCountStaged }}</span>
-              <Diff :value="resourcesCountDiff" class="font-weight-bold">
-                <template slot-scope="{ sign, value }">
-                  ({{ sign }}{{ value ? value : '-' }})
+            </VLayout>
+            <div style="margin-left: -24px;">
+              <StudioTree
+                :treeId="stagingId"
+                :nodeId="stagingId"
+                :selectedNodeId="nodeId"
+                :onNodeClick="onTreeTopicClick"
+                :root="true"
+              />
+            </div>
+          </ResizableNavigationDrawer>
+
+          <VContainer fluid class="pa-0 ma-0" :style="{ backgroundColor: 'white' }">
+            <VToolbar v-if="breadcrumbsItems.length" dense color="transparent" flat>
+              <Breadcrumbs :items="breadcrumbsItems" class="pa-0">
+                <template #item="props">
+                  <span
+                    class="notranslate"
+                    :class="[props.isLast ? 'font-weight-bold text-truncate' : 'grey--text']"
+                  >
+                    {{ props.item.title }}
+                  </span>
                 </template>
-              </Diff>
-            </span>
-            <span class="pl-2" data-test="bottom-bar-stats-file-size">
-              {{ $tr('totalSize') }}:
-              <span class="font-weight-bold">{{ formatFileSize(fileSizeStaged) }}</span>
-              <Diff :value="fileSizeDiff" class="font-weight-bold">
-                <template slot-scope="{ sign, value }">
-                  ({{ sign }}{{ value ? formatFileSize(value) : '-' }})
+              </Breadcrumbs>
+            </VToolbar>
+
+            <VLayout>
+              <VList
+                shrink
+                class="pa-0"
+                :style="{width: '100%', backgroundColor: $vuetify.theme.backgroundColor}"
+              >
+                <template v-for="child in children">
+                  <ContentNodeListItem
+                    :key="child.id"
+                    :node="child"
+                    :compact="isCompactViewMode"
+                    data-test="node-list-item"
+                    @infoClick="goToNodeDetail(child.id)"
+                    @topicChevronClick="goToTopic(child.id)"
+                    @click.native="onNodeClick(child)"
+                    @dblclick.native="onNodeClick(child)"
+                  />
                 </template>
-              </Diff>
-            </span>
-          </VFlex>
-
-          <VFlex class="bottom-bar-btns">
-            <VDialog
-              v-model="displaySummaryDetailsDialog"
-              width="500"
-            >
-              <template v-slot:activator="{ on }">
-                <VBtn
-                  flat
-                  data-test="display-summary-details-dialog-btn"
-                  v-on="on"
+              </VList>
+
+              <ResourceDrawer
```

Comment:
Should the resource drawer extend over the breadcrumbs as well? It seems off to have the drawer border not connect with the toolbar border
![image](https://user-images.githubusercontent.com/7447496/88349466-045dad00-cd05-11ea-93bf-75c0a11c5e10.png)